admin_manual/installation/source_installation.rst

```diff
@@ -14,7 +14,7 @@ archive <https://nextcloud.com/install/>`_. This method is recommended to instal
 
 
 If you prefer a more automated installation of Nextcloud and there are no packages for your Linux distribution, you have the option to
-install the community `Snap Package <https://snapcraft.io/nextcloud>`_. This includes a full production-ready stack, will maintain your HTTPS certificates for you, and will automatically update as needed to stay secure. You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server. It will setup everything for you and include scripts for automated installation of apps like; Collabora, OnlyOffice, Talk and so on. Please note that those two options are not officially supported by Nextcloud GmbH.
+install the community `Snap Package <https://snapcraft.io/nextcloud>`_. This includes a full production-ready stack, will maintain your HTTPS certificates for you, and will automatically update as needed to stay secure. You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server or `NextcloudPi scripts <https://nextcloudpi.com/>`_ (dtto atop Debian). It will setup everything for you and include scripts for automated installation of apps like; Collabora, OnlyOffice, Talk and so on. Please note that those three options are not officially supported by Nextcloud GmbH.
```

Comment:
dtto atop Debian? :thinking:

Reply:
Tipo (sorry for that) - should be ditto (https://en.wikipedia.org/wiki/Ditto) and "atop" as used for example here https://www.phoronix.com/news/Linux-Mint-LMDE5-Released. So in meaning "basically very similar thing, but based on Debian".

Reply:
OK. Would you mind sending another pull request to fix the typo?

Reply:
Yes, for sure - see https://github.com/nextcloud/documentation/pull/9473